include/deal.II/matrix_free/hanging_nodes_internal.h

```diff
@@ -13,27 +13,75 @@
 //
 // ---------------------------------------------------------------------
 
-#ifndef dealii_cuda_hanging_nodes_internal_h
-#define dealii_cuda_hanging_nodes_internal_h
+#ifndef dealii_hanging_nodes_internal_h
+#define dealii_hanging_nodes_internal_h
 
 #include <deal.II/base/config.h>
 
-#ifdef DEAL_II_COMPILER_CUDA_AWARE
-
-#  include <deal.II/base/cuda_size.h>
-#  include <deal.II/base/utilities.h>
+#include <deal.II/base/cuda_size.h>
+#include <deal.II/base/utilities.h>
 
-#  include <deal.II/dofs/dof_accessor.h>
-#  include <deal.II/dofs/dof_handler.h>
+#include <deal.II/dofs/dof_accessor.h>
+#include <deal.II/dofs/dof_handler.h>
 
-#  include <deal.II/fe/fe_q.h>
-#  include <deal.II/fe/fe_tools.h>
+#include <deal.II/fe/fe_q.h>
+#include <deal.II/fe/fe_tools.h>
 
 DEAL_II_NAMESPACE_OPEN
 
+#ifdef DEAL_II_COMPILER_CUDA_AWARE
 namespace CUDAWrappers
 {
   namespace internal
+  {
+    namespace internal
+    {
```

Comment:
Is there a reason to have two nested `internal` namespaces? It seems a bit confusing at least.